notebooks/monitoring.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+echo ==================================
+echo =========== MONITORING ===========
+echo ==================================
+echo --- General Information ---
+# number of cores
+echo \#CPU: $(nproc)
+
+# multiply by 10^-6 to convert KB to GB
+echo Total Memory: $(cat /proc/meminfo | grep MemTotal | awk 'BEGIN { FS=" " } ; { print $2/1000000 }')G
+echo Total Disk space: $(df -h | grep cromwell_root | awk '{ print $2}')
+echo
+
+
+function runtime_info() {
+        echo [$(date)]
+        echo \* CPU usage: "$(get_cpu_usage)"%
+        echo \* Memory usage: "$(get_mem_usage)"%
+        echo \* Disk usage: $(df | grep cromwell_root | awk '{ print $5 }')
+}
+
+
+function get_cpu_usage() {
+        # get the cpu usage since a given time (w/o idle or iowait)
+        # user+nice+system+irq+softirq+steal
+        cpu_used_cur=$(cat /proc/stat | grep "cpu " | awk 'BEGIN { FS=" " } ; { print $2+$3+$4+$7+$8+$9 }')
+
+        # get the total cpu usage since a given time (including idle and iowait)
+        # user+nice+system+idle+iowait+irq+softirq+steal
+        cpu_total_cur=$(cat /proc/stat | grep "cpu " | awk 'BEGIN { FS=" " } ; { print $2+$3+$4+$5+$6+$7+$8+$9 }')
```

Comment:
Everything up to the `print` is common with the line above, could you pull it into its own function?